src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -57,6 +63,18 @@ class WidgetController(
             "additionalClass" to ""
         )
 
+        try {
```

Comment:
This try catch block could be moved ("hidden") inside the `isIncompatibleOSVersion` method where the `catch` just returns `true`. This would remove the duplicate widget rendering return. 
Second option would be using `finally` instead of `catch` and removing the second return.